front/src/applications/operationalStudies/views/Scenario.tsx

```diff
@@ -228,13 +227,16 @@ export default function Scenario() {
                     displayTrainScheduleManagement={displayTrainScheduleManagement}
                     setDisplayTrainScheduleManagement={setDisplayTrainScheduleManagement}
                     infraState={infra.state}
+                    refetchTimetable={refetchTimetable}
```

Comment:
You don't need this function to update the timetable list when adding a train. You could just add the `timetable` tag in `invalidatesTags` of the `postTrainScheduleStandaloneSimulation` endpoint.

Reply:
See my comment above :top: